__performance__/bin/commands/compare.js

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2024 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+const fs = require('node:fs');
+const path = require('node:path');
+const { diffJson } = require('diff');
+const {
+  green, red, dim, underline,
+} = require('colorette');
+const math = require('../util/math');
+const pick = require('../util/pick');
+const generateTable = require('../util/metricsTable');
+const coerceTestId = require('../util/coerceTestId');
+const {
+  diffProcessor,
+  pValueProcessor,
+  variantProcessor,
+} = require('../util/processors');
+const banner = require('../util/banner');
+const { colors } = require('../util/format');
+const options = require('../util/options');
+
+module.exports.command = 'compare';
+
+module.exports.describe = 'Compare two performance test results';
+
+module.exports.handler = async function compare(argv) {
+  const { italic } = colors();
+  const cData = {
+    ...JSON.parse(await fs.promises.readFile(path.join(argv.control.dir, 'metrics-k6.json'))),
+    ...JSON.parse(await fs.promises.readFile(path.join(argv.control.dir, 'metrics-prom.json'))),
+  };
+  const vData = {
+    ...JSON.parse(await fs.promises.readFile(path.join(argv.variant.dir, 'metrics-k6.json'))),
+    ...JSON.parse(await fs.promises.readFile(path.join(argv.variant.dir, 'metrics-prom.json'))),
+  };
```

Comment:
```suggestion
 const [controlk6, controlProm, variantk6, variantProm] = await Promise.all([
    fs.promises.readFile(path.join(argv.control.dir, 'metrics-k6.json')),
    fs.promises.readFile(path.join(argv.control.dir, 'metrics-prom.json')),
    fs.promises.readFile(path.join(argv.variant.dir, 'metrics-k6.json')),
    fs.promises.readFile(path.join(argv.variant.dir, 'metrics-prom.json'))
   ]);
  const cData = {
    ...JSON.parse(controlk6),
    ...JSON.parse(controlProm),
  };
  const vData = {
    ...JSON.parse(variantk6),
    ...JSON.parse(variantProm),
  };
```

Small performance boost and slightly more readable